library/alloc/src/slice.rs

```diff
@@ -494,6 +566,29 @@ impl<T> [T] {
         hack::to_vec(self, alloc)
     }
 
+    /// Tries to copy `self` into a new `Vec` with an allocator.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(allocator_api)]
+    ///
+    /// use std::alloc::System;
+    ///
+    /// let s = [10, 40, 30];
+    /// let x = s.try_to_vec_in(System).unwrap();
+    /// // Here, `s` and `x` can be modified independently.
+    /// ```
+    #[inline]
+    #[unstable(feature = "kernel", issue = "none")]
```

Comment:
Wasn't sure what feature/issue to use.

Reply:
The feature gate should be something like `more_fallible_allocation_methods`; anything that hasn't been used before. You'll need to add that feature-gate name to the list of feature gates, along with the tracking issue and target version for stabilization.

The issue gets filed after preliminary review of the PR; file a new GitHub issue and use the "library tracking issue" template.

Reply:
> The issue gets filed after preliminary review of the PR; file a new GitHub issue and use the "library tracking issue" template.

I interpreted this to mean it has by now already had preliminary review review, so I should go ahead and make the issue right  way. Sorry if I was mistaken!

Issue is #86942.